src/mainwindow.cpp

```diff
@@ -651,6 +656,11 @@ void MainWindow::UpdateShopMenu() {
         title += " [" + Util::StringJoin(app_->shop().threads(), ",") + "]";
     ui->actionForum_shop_thread->setText(title.c_str());
     ui->actionAutomatically_update_shop->setChecked(app_->shop().auto_update());
+
+    std::string action_label = "Remote process list script";
+    if (auto_online_.IsRemoteScriptSet())
+        action_label += " [******]";
```

Comment:
Also `[******]` is supposed to be used for sensitive data, putting the script path here wouldn't hurt I think.